apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -466,7 +463,48 @@ const BookingPage = ({
                     </div>
                     <EventTypeDescriptionSafeHTML eventType={eventType} />
                   </div>
-                )}
+                  <EventTypeDescriptionSafeHTML eventType={eventType} />
+                </div>
+              )}
+              {eventType?.locations && (
+                <>
+                  {(eventType.locations as string).length === 1 && (
```

Comment:
Why do we need type casting here? We try to avoid that unless strictly necessary

Reply:
If there isnt any type casting, I end up getting type errors saying that `type 'type' is not assignable to type 'other type'`

Reply:
This means that it should be fixed upper in the stream